### DISCUSSION TITLE
Different acctions mode browsing through my list

### DISCUSSION THREAD

**Discussion by infiniteroles** [Laravel-Backpack/community-forum#999] [Q&A (Help)]
Good morning,

I am experiencing something very curious while browsing through my lists. In some cases, instead of displaying the compressed list of actions in a dropdown, it shows them as links. It is strange because it happens within the same list, and I don't understand where this behavior might be coming from. Does this sound familiar to you? In the screenshot, I show you exactly what I mean.
![screen](https://github.com/Laravel-Backpack/community-forum/assets/11851266/26d94e78-6be6-4a40-83d7-b29f90f61ce2)

Thank you very much,
Oscar

**Comment by jcastroa87** [comment]
Hello @infiniteroles,

Thank you for reaching out.

Did all the rows have the same number of options? Also, did you follow our [documentation](https://backpackforlaravel.com/docs/6.x/crud-operation-list-entries#merging-line-buttons-into-a-dropdown) to force the display as a dropdown?

Please let me know if this resolves your issue.

Cheers.

**Reply by jcastroa87** [reply]
Hello @infiniteroles please let me know if this worked for you.

Thanks.

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or created a new one if needed.

Cheers.
